taskcluster/ci/build/kind.yml

```diff
@@ -130,16 +130,6 @@ job-defaults:
 
 overriden-jobs:
     # Treeherder group names are too long to fit in 25 chars
-    browser-engine-gecko-nightly:
-        treeherder:
-            symbol: 'browser-engine-gecko-nigh(B)'
-    nightly-browser-engine-gecko-nightly:
-        treeherder:
-            symbol: 'browser-engine-gecko-nigh(BN)'
-    release-browser-engine-gecko-nightly:
-        treeherder:
-            symbol: 'browser-engine-gecko-nigh(BR)'
-
```

Comment:
@pocmo , in this file we don't need anything related to gecko, the previous changes were only related to gecko nightly?

Reply:
In this file we only needed to remove `-beta` and `-nightly` entries.

Reply:
👍🏽